lab_1_keywords_tfidf/main.py

```diff
@@ -12,12 +14,21 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
     Parameters:
     text (str): Original text
 
-    Returns:
+    Returns:  
     list[str]: A sequence of lowercase tokens with no punctuation
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    for punctuation in string.punctuation:
+        if punctuation in text:
+            text = text.strip()
+            text = text.replace('-', '')
+            text = text.replace('.', '')
```

Comment:
why do you need to replace . and - with empty string on each of the iterations? 
aren't those symbols included in string.punctuation?

Reply:
yes, but for some incomprehensible reason, it doesn't remove it everywhere